test/controllers/access_requests_controller_test.rb

```diff
@@ -55,7 +55,9 @@
       describe 'enabled' do
         let(:manager_email) { 'manager@example.com' }
         let(:reason) { 'Dummy reason.' }
-        let(:request_params) { {manager_email: manager_email, reason: reason, project_ids: Project.all.pluck(:id)} }
+        let(:role) { Role::DEPLOYER }
```

Comment:
And on the tests, just to keep it consistent, same thing: we need to change it to ProjectRole.